front/planning.php

```diff
@@ -37,7 +37,9 @@
 
 include ('../inc/includes.php');
 
-Session::checkRight("planning", READ);
+if (!isset($_GET['genical'])) {
```

Comment:
Are we sure this could not cause any user to read contents even if he does not have correct ACLs, just adding the parameter in the URL?